doc/world_format.txt

```diff
@@ -521,6 +521,10 @@ Object types:
     s32 velocity.y * 10000
     s32 velocity.z * 10000
     s32 yaw * 1000
+    if PROTOCOL_VERSION >= 37:
+      u8 version2 (=1)
```

Comment:
please also document that the version above is always kept at 1

Reply:
Done.

Reply:
That's not *exactly* what he meant. `version` is and must always stay `1` for backwards compatibility.
Suggestion: `u8 compatibility_byte (always 1)` even though the names don't match.

Reply:
But does `version` have to stay `1` in the world file? Saving backwards-compatible makes sense if you are sending the data to a client but not if you are saving to file.
I will just take your suggestion as I'm surely wrong.

Reply:
Network:
https://github.com/minetest/minetest/blob/86d7f84b899a507e979f1845f2057cce6f84e743/src/content_sao.cpp#L558
Map:
https://github.com/minetest/minetest/blob/86d7f84b899a507e979f1845f2057cce6f84e743/src/content_sao.cpp#L613-L620

`version(1)` was implemented in a forwards incompatible manner, resulting in unknown entities when loading the world from an old client. 5.x has to keep the map compatibility, hence a new version byte was needed to work around this issue.